js/packages/web/src/components/ArtContent/index.tsx

```diff
@@ -190,14 +191,42 @@ const HTMLContent = ({
       ? files[0]
       : animationUrl;
   return (
-    <iframe
-      allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-      sandbox="allow-scripts"
-      frameBorder="0"
-      src={htmlURL}
-      className={className}
-      style={style}
-    ></iframe>
+    <div
+      style={{
+        paddingTop: '100%',
+        position: 'relative',
+      }}
+    >
+      {!loaded ? (
```

Comment:
It can be just like this:
```suggestion
      {!loaded && (
```
```suggestion
      {!loaded ? (
```